web/packages/teleport/src/AppLauncher/AppLauncher.test.tsx

```diff
@@ -36,42 +36,42 @@ const launcherPathTestCases: {
   {
     name: 'no state and no path',
     path: '?path=',
-    expectedPath: 'x-teleport-auth',
+    expectedPath: 'undefined/x-teleport-auth',
```

Comment:
Why would we ever expect "undefined" in a URL?

Reply:
I originally misunderstood this test case, but this code is not included anymore so N/A